connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -39,6 +39,38 @@ package object dsl {
               .addAllParts(identifier.asJava)
               .build())
           .build()
+
+      def protoQualifiedAttr: proto.Expression.QualifiedAttribute =
+        proto.Expression.QualifiedAttribute.newBuilder()
+          .setName(identifier.mkString("."))
```

Comment:
For LocalRelation's attributes, how to convert `identifier: Seq[String]` to it? 

Do we need to add an extra `qualifier: Seq[String]`?

Reply:
actually `a.b` should not directly map to `QualifiedAttribute`. It can be `GetStructField(QualifiedAttribute...)`

Reply:
I think we can follow the catalyst dsl
1. if we call `protoAttr`, then we parse the name (split by dot) and create `UnresolvedAttribute`
2. if we call `bool`, then we directly use the name as attribute name and create `AttributeReference`

Reply:
ah let me take a look.